vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow/src/main/java/com/vaadin/flow/component/richtexteditor/RichTextEditor.java

```diff
@@ -207,6 +218,16 @@ public String getHtmlValue() {
         return sanitize(htmlValueString);
     }
 
+    /**
+     * HTML representation of the rich text editor content.
+     *
+     * @return the {@code htmlValue} property from the webcomponent
+     */
+    @Synchronize(property = "htmlValue", value = "html-value-changed")
+    protected String getHtmlValueString() {
```

Comment:
- Could be marked private since it's only used internally.
- Alternatively, we could remove this method and move the `@Synchronize` annotation to the public getHtmlValue method instead

Reply:
Marked as private.